brave/browser-laptop
# Converts BraveryPanel into redux component

Submitter Checklist:


Resolves #9454

Auditors: @bsclifton @bridiver

Test Plan:
- `npm run test -- --grep='Bravery Panel'`
- check if bravery panel is opened when you click on lion head
- check if you can toggle all checkboxes
- check if blocking numbers are correct


Reviewer Checklist:

Tests
